roborock/devices/traits/b01/q7/__init__.py

```diff
@@ -28,23 +28,27 @@ def __init__(self, channel: MqttChannel) -> None:
         """Initialize the B01Props API."""
         self._channel = channel
 
-    async def query_values(self, props: list[RoborockB01Props]) -> B01Props | None:
-        """Query the device for the values of the given Q7 properties."""
-        result = await send_decoded_command(
+    async def send(self, command: RoborockB01Q7Methods, params: dict) -> Any:
```

Comment:
Is this intended to be publicly called by clients? If no, then `_send`. If yes then add a pydoc string.

Should this be the first function the user reads about in this file or should we put it below all the higher level API calls?

Reply:
I don't have a preference. Right now this has a single API for the entire protocol which is fine with me if you wan tto keep it here. Then just add the pydoc string, and i'd still suggest moving it lower in the file.